flake.nix

```diff
@@ -0,0 +1,25 @@
+{
+  description = "doorctl";
```

Comment:
What's doorctl, is this copied over from another project?

Reply:
Oops, yes.